extensions/samples/electron-webpack/README.md

```diff
@@ -0,0 +1,52 @@
+## About
+This is a sample that depicts how to integrate the `msal-node-extensions` library to your [electron](https://www.electronjs.org/) application that has been bundled by [webpack](https://webpack.js.org/).
+
+## Getting started
+To get started using this sample, you need to do the following:
+
+**1. Change the MSAL Electron Configuration** \
+Navigate to the `main.ts` file in the src folder and replace the `<CLIENT_ID>` with your client ID. Follow this [steps](https://docs.microsoft.com/en-us/graph/toolkit/get-started/build-an-electron-app#add-new-application-registration-in-azure-ad-to-get-a-client-id) to generate the client ID from Azure.
```

Comment:
For app registration, can we use our docs here instead, for instance https://docs.microsoft.com/azure/active-directory/develop/scenario-desktop-app-registration

Reply:
Or is the sample meant to be used with Microsoft Graph Toolkit with Electron

Reply:
Yes, to make the sample more wholesome, I decided to incorporate Graph Toolkit to the Electron app.